cmd/humioctl/alerts_export.go

```diff
@@ -43,20 +43,20 @@ func newAlertsExportCmd() *cobra.Command {
 
 			alert, apiErr := client.Alerts().Get(view, alertName)
 			if apiErr != nil {
-				cmd.Println(fmt.Errorf("Error fetching alert: %s", apiErr))
+				cmd.Println(fmt.Errorf("error fetching alert: %s", apiErr))
```

Comment:
I don't think there's any purpose in `fmt.Errorf` here, we might as well change to `cmd.Printf` and keep the capitalized version.

Reply:
Right. That's fair. Let's go with that.